website/server/controllers/api-v3/members.js

```diff
@@ -402,6 +363,25 @@ function _getMembersForItem (type) {
   };
 }
 
+async function _getMemberTasksFromChallenge (challenge, member, user, res) {
```

Comment:
since this is only used by one route I don't think it makes sense to have it an separate method, unless we can merge this and `getMembersTasksForChallenge` in a single one